### ISSUE TITLE
How to select USSD menu

### ISSUE THREAD

**Issue by shuhail555** [gammu/gammu#173]
I was trying to use USSD using gammu. [ gammu --getussd *111# ] works fine. But in my case it returns with 5 options & I need to reply with a number.

So is that possible to send reply to any ussd response using gammu CLI.

**Comment by nijel** [comment]
Currently this is not implemented in Gammu CLI, but there is python-gammu sample script to do that: https://github.com/gammu/python-gammu/blob/master/examples/service_numbers.py

**Comment by ktngoykalolo** [comment]
what would it take to include USSD response support on the codebase ?

**Comment by nijel** [comment]
It should not be hard, there is already [getussd](https://wammu.eu/docs/manual/gammu/#cmdoption-gammu-arg-getussd), which can do one time operation, you just need to check whether response is expected and prompt for further user input in that case.

**Comment by ktngoykalolo** [comment]
would that be enough to manage a whole interaction during a USSD session ?

**Comment by nijel** [comment]
Yes, that should work just fine (it works from Python bindings, so there is no reason why it would not work here).
